deploy/role_binding.yaml

```diff
@@ -3,9 +3,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: compute-node-operator
 subjects:
-- kind: ServiceAccount
-  name: compute-node-operator
-  namespace: openshift-machine-api
+  - kind: ServiceAccount
```

Comment:
ditto

Reply:
yeah was done automatically by my editor, reverted it